lib/src/us/kbase/kbasefeaturevalues/ExpressionMatrix.java

```diff
@@ -328,9 +348,79 @@ public void setAdditionalProperties(java.lang.String name, Object value) {
         this.additionalProperties.put(name, value);
     }
 
+    @Override
```

Comment:
For future reference, eclipse (and probably other IDEs) can generate equals and hashcode for you